src/hivpy/sex_behaviour_data.py

```diff
@@ -1,6 +1,19 @@
 import numpy as np
 import scipy.stats as stat
 
+from .demographics import SexType
+
+# Initial sexual behaviour group probabilities
+# Taken from the SAS code lines 1249 and 1283
```

Comment:
```suggestion
# Taken from the SAS code Section 2
```
(who knows when line numbers will change!)

Reply:
I did think about this, and happy to change that over. I'm only intending for this to be a temporary comment as at some point either the model doc - or hivpy itself - will become the authority!